libvisual-plugins/plugins/actor/dancingparticles/font.cpp

```diff
@@ -0,0 +1,229 @@
+#include "font.h"
+#include "etoile.h"
+#include "libvisual/lv_log.h"
+#include "libvisual/lv_video.h"
+#include <cstring>
+#include <algorithm>
+#include <limits>
+#include <vector>
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+#include FT_GLYPH_H
+
+const char *fontPath = DATA_DIR "/font.pfb";
+const int   fontSize = 25; // Does this match the T1lib size parameter?
```

Comment:
My vote for dropping that comment now, and renaming the variable to `fontSizePoints` or similar to communicate the unit behind the number.

Reply:
I agree with dropping the comment. It should be renamed `fontSizePixels` though.

Reply:
@kaixiong right! Maybe a new comment that these are about the EM square would help the next developer, just an idea.